src/Symfony/Component/Scheduler/RecurringMessage.php

```diff
@@ -75,14 +75,15 @@ public static function trigger(TriggerInterface $trigger, object $message): self
             return new self($trigger, $message);
         }
 
-        $description = '';
-        try {
-            $description = $message instanceof \Stringable ? (string) $message : serialize($message);
-        } catch (\Exception) {
+        $description = $message::class;
+        if ($message instanceof \Stringable) {
+            try {
+                $description .= " ($message)";
+            } catch (\Exception) {
+            }
         }
-        $description = sprintf('%s(%s)', $message::class, $description);
 
-        return new self($trigger, new StaticMessageProvider([$message], $description));
+        return new self($trigger, new StaticMessageProvider([$message], strtr(substr(base64_encode(hash('xxh128', serialize($message), true)), 0, 7), '/+', '._'), -7), $description));
```

Comment:
FYI, there was a syntax error on this line. I fixed that in a45f6cbeb6e05feb24b8d67e6f361da418511d3b. Please double-check that I didn't mess anything with the logic implemented here.